packages/dashboard/src/components/internalDashboard/keyboardShortcuts.ts

```diff
@@ -45,28 +44,21 @@ export const useKeyboardShortcuts = () => {
     dispatch(onSendWidgetsToBackAction());
   };
 
-  const deleteWidgets = useCallback(() => {
```

Comment:
I think instead of moving this out of the keyboard shortcuts, we can either return the modal component to be added to whatever component this hook is being used in, or we can pass in a ref to the hook which we can attach the modal to.